ebpf-io/ebpf.io-website
# feat: update link

**This PR:**
Updates the link to the store.
Adds eBPF Summit to Events.

**Steps to test**

- Open the [Swag page](https://deploy-preview-427--ebpfio.netlify.app/swag/)
- Open the [Events page](https://deploy-preview-427--ebpfio.netlify.app/events/)
- Confirm that everything looks and works as expected
